client/js/screens/FindLocker/FindLocker.js

```diff
@@ -1,10 +1,19 @@
 import React from "react";
-import { Text, View } from "react-native";
+import { Text, View, Button } from "react-native";
+import { withNavigation } from "react-navigation";
+import Locker from "../Locker";
 
-const FindLocker = ({ params }) => (
+const FindLocker = ({ navigation }) => (
   <View>
     <Text>This is the Find Locker Screen</Text>
+    <Button
+      onPress={() => {
+        navigation.push("Locker");
+      }}
+      title="Locker Info"
+      color="#841584"
+    />
   </View>
 );
 
-export default FindLocker;
+export default withNavigation(FindLocker);
```

Comment:
use navigation as a prop (pass it from the container), not from withNavigator, for the screens.

Reply:
Please resolve this issue.